src/Client/Web/Bit.Client.Web.BlazorUI/Components/Pickers/BitDatePicker.razor.cs

```diff
@@ -86,6 +99,8 @@ public bool IsOpen
 
         [Parameter] public Func<BitDate,string> OnSelectDate { get; set; }
 
+        [Parameter] public EventCallback<string?> ValueChanged { get; set; }
```

Comment:
the type parameter of the EventCallback should be exactly the same as the main parameter's type itself in here.
so please change it from `string?` to `string`  or change the type of Value parameter to `string?`

Reply:
and also add a private field name `ValueHasBeenSet`
it is our main convention for the 2-way bound parameters. otherwise, the source generated by the source-generator will fail at build.